src/python/ensembl/io/genomio/genome_metadata/dump.py

```diff
@@ -147,6 +148,29 @@ def check_assembly_version(gmeta_out: Dict[str, Any]) -> None:
             raise ValueError(f"Assembly version is not an integer in {assembly}")
 
 
+def check_genebuild_version(gmeta_out: Dict[str, Any]) -> None:
+    """Update the genebuild version if not set.
+    Get the version from the genebuild id (and remove that value if any).
+
+    Args:
+        gmeta (Dict[str, Any]): Nested metadata key values from the core metadata table.
```

Comment:
```suggestion
        gmeta: Nested metadata key values from the core metadata table.
```
This is something we want to avoid, particularly with the new `mkdocs` system.